src/js/components/TaskDetail.js

```diff
@@ -27,14 +32,14 @@ const METHODS_TO_BIND = [
   'onTaskDirectoryStoreSuccess'
 ];
 
-class TaskSidePanelContents extends SidePanelContents {
+class TaskDetail extends mixin(InternalStorageMixin, TabsMixin, StoreMixin) {
   constructor() {
     super(...arguments);
 
     this.tabs_tabs = Object.assign({}, TABS);
 
     this.state = {
-      currentTab: Object.keys(this.tabs_tabs).shift(),
+      currentTab: 'details',
```

Comment:
Interesting, that is new to me. Do you have a link for that? 
I thought that's why a Map was introduced. To ensure the order.
But we use an Object here, not a Map. So the order is not guaranteed. I am pretty sure.